bitcoin/src/blockdata/transaction.rs

```diff
@@ -504,6 +504,14 @@ impl fmt::UpperHex for Sequence {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result { fmt::UpperHex::fmt(&self.0, f) }
 }
 
+impl fmt::Debug for Sequence {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        f.write_str("Sequence(0x")?;
+        fmt::LowerHex::fmt(self, f)?;
```

Comment:
This will mess up formatting. I'd just use `write!(f, "Sequence(0x{:08x})")` or write it into a temporary `ArrayString` and format that one.

Reply:
Face palm, when writing this change I literally thought to myself "now what did Kix tell me last time when doing `fmt` stuff".

> but isn't that exactly what they're asking for?

I rekon because we get the "Sequence(...)" text we don't want to have control over padding etc.

Reply:
We want `println!("{:020?}", seq)` to print "        Sequence(0xffffffff))", right? not "Sequence(          0xffffffff)" which it does now (untested).

Reply:
Exactly.

Reply:
I'm now not sure which behaviour is correct but I think its not super important so leaving it as `write!(f, "Sequence({:#010x)")`